content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -114,6 +119,13 @@ const NewTabPage = React.createClass({
               length={HIGHLIGHTS_LENGTH} sites={props.Highlights.rows}
               prefs={props.Prefs.prefs} />
           }
+          {showBookmarks &&
+            <section>
```

Comment:
this wrapper `<section/>` isn't really needed. I removed it from the other components above.